music_assistant/server/providers/spotify/__init__.py

```diff
@@ -229,8 +229,35 @@ async def get_library_tracks(self) -> AsyncGenerator[Track, None]:
             if item and item["track"]["id"]:
                 yield await self._parse_track(item["track"])
 
+    async def _get_liked_songs_playlist(self) -> Playlist:
+        liked_songs = Playlist(
+            item_id="tracks",
+            provider=self.domain,
+            name="Liked Songs",  # TODO to be translated
+            owner="Me",  # TODO Get logged in user display name
+            provider_mappings={
+                ProviderMapping(
+                    item_id="tracks",
+                    provider_domain=self.domain,
+                    provider_instance=self.instance_id,
+                    url="https://open.spotify.com/collection/tracks",
+                )
+            },
+        )
+
+        liked_songs.is_editable = False  # TODO Editing requires special endpoints
+
+        liked_songs.metadata.images = [
```

Comment:
don't forget to set the metadata's checksum field otherwise it wont be refreshed on updates.

Reply:
Liked Songs do not have anything like `snapshot_id` in regular playlists. It's really just a plain list of tracks. Checking how other providers are doing it and it's applicable here

Reply:
ah in that case just assign a random value like time.time() or a random integer or whatever to invalidate the cache

Reply:
Done